frontend/src/routes/account-settings/+page.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
```

Comment:
Right now we have an `/account` and an `/account-settings` page. They look to be the same thing, except `account-settings` has a logout button.